mediapipe/tasks/python/benchmark/vision/image_classifier/image_classifier_benchmark.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 The MediaPipe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Benchmark for the image classifier task."""
+import argparse
+import time
+import numpy as np
+import mediapipe as mp
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+
+_IMAGE_FILE = 'burger.jpg'
+
+
+def run(model: str, n_iterations: int, delegate: python.BaseOptions.Delegate):
+    """Run asynchronous inference on images and benchmark.
```

Comment:
Please document the return value (and maybe allow us to pass in the percentile to use?)

Reply:
I've fixed it now.